_data/current.yml

```diff
@@ -16,7 +16,14 @@
   dates: "February 13-14, 2025"
   url: https://robocon.io/?utm_source=testingconferences
   status: CFP is Closed
-
+  
+- name: Continuous Innovation: Mastering DevOps & Testing
```

Comment:
hi @Kumar2035 you cant have colons in the name. If you want them, put quotes around the whole name. Thanks